packml_msgs/msg/ItemizedStats.msg

```diff
@@ -0,0 +1,6 @@
+# This message contains itemized statics for PackML statistics reporting
+# Each item containds a unique
+
+int16 id                          # unique identifier (user defined)
+int32 count                       # number of occurances
+std_msgs/Duration duration        # duration of in error state
```

Comment:
I'm not completely familiar with how PackML defines all of this, but it seems strange that a generic `ItemizedStats` message (with a user defined `id` field) would contain a field especially for `duration of in error state`?